### PR TITLE
stream9 without ansible-builder (via @shanemcd)

### PR DESCRIPTION
**Pull request by relrod**
@shanemcd this is your branch, dusted off with some minor changes and tested briefly on my AWX instance.

What would it take to eventually push this through?

### CODE REVIEW

**Comment by relrod** [comment]
~~I guess the first step would be getting CI to avoid using ansible-builder for now too :rofl:~~

Done, modified tox.ini to just use docker/podman build for now.

**Review by shanemcd — APPROVED** [review]
Thanks for dragging this across the finish line!

File: _build/requirements.yml [hunk 1]
```diff
@@ -7,7 +7,9 @@ collections:
   - name: google.cloud
   - name: openstack.cloud
   - name: community.vmware
-  - name: ovirt.ovirt
+  - name: https://github.com/shanemcd/ovirt-ansible-collection.git
```
Comment (kurokobo) [inline _build/requirements.yml]: Is it okay that referring collections from personal repo?

Comment (oyvindkolbu) [reply _build/requirements.yml]: The patch in the private fork is now in master at https://github.com/oVirt/ovirt-ansible-collection.git, so could use that instead.

Comment (relrod) [reply _build/requirements.yml]: Thanks, will get this PR updated before I merge it.

**Comment by relrod** [comment]
I'm going to merge this - I expect it is going to break some people because we're updating ansible and the centos stream version, both.

I'm writing this comment here so we can link people to it later:

If this upgrade breaks your playbooks, you have a few options.
- Update your playbooks to work on newer ansible-core (preferred)
- Use `quay.io/ansible/awx-ee:21.11.0` which was tagged during last AWX release and still contains the older stream 8 image

**Pull request merged**